### ISSUE TITLE
Background player resets and doesn't play content when disconnecting and reconnecting Bluetooth unless closed and opened again

### ISSUE THREAD

**Issue by captaincupcakes** [TeamNewPipe/NewPipe#5560]
### Checklist

- [x] I am using the latest version - x.xx.x 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

Play content on background mode on connected device (mostly happens in cars and home theaters),disconnect and connect again

### Actual behaviour
 I have noticed that every time,in any car,I disconnect and connect to the phone bluetooth the background player "resets" (kind of like goes back to zero) and I'm unable to play whatever content I had playing unless I close the background player and start it again

This bug has gone back all the way from 0.17.3 (when I found out about newpipe) until now,and it does it mostly on cars with Bluetooth and home theaters with Bluetooth as well.
Also device doesn't matter,as my xperia 5 II,pixel ii and galaxy s8 all did the same thing.

### Expected behavior
It would be nice if somebody has a look into this bug,that way whenever I disconnect and  reconnect to Bluetooth,I can still play my content without  having to close and open again the background player
Attached is a picture of the result of disconnecting and reconnecting again (pressing play doesn't do anything)
![Screenshot_20210211-101835](https://user-images.githubusercontent.com/78338675/107727487-71dd2500-6cb9-11eb-9f16-21f0703e188c.png)

### Logs

the error doesn't generate any logs that I can see

### Device info

 - Android version/Custom ROM version:android 10 58.0.A.28.51
 - Device model:sony xperia 5 II XQ-AS62

**Comment by Ammako** [comment]
The same thing causes newpipe to show up with an [Unknown] notification when using bluetooth with something unrelated to newpipe. Have to open newpipe every time and then close it for the empty notification to go away.

**Comment by captaincupcakes** [comment]
Yes to that as well... earlier versions used to do that when I connected,
although since they improved the background player it stopped doing it for
me.

**Comment by captaincupcakes** [comment]
Did someone got to look at this???
